float64 x
float64 y
float64 z
string type
---
bool response